### PR TITLE
Email activation

### PR DESCRIPTION
**Pull request by chan227**
User need to activate himself by using his own email link now.

### CODE REVIEW

**Comment by chan227** [comment]
ready

File: app/helpers/account_activations_helper.rb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+module AccountActivationsHelper
```
Comment (ngocvt-0484) [inline app/helpers/account_activations_helper.rb]: remove empty file

Comment (chan227) [reply app/helpers/account_activations_helper.rb]: done

**Comment by chan227** [comment]
ready

**Pull request merged**